src/components/RenameHeader.tsx

```diff
@@ -97,7 +97,6 @@ const RenameHeader: FC<Props> = ({
                 <li
                   className="p-breadcrumbs__item name u-truncate"
                   onClick={enableRename}
-                  title={name}
```

Comment:
We use the title attribute frequently, because if the content is very long (i.e. a very long instance name), the text will be truncated. Adding the title allows hovering to get a tooltip with the full, non-truncated name of the resource.

Reply:
As discussed, will keep all `title` attributes and adjust them to have slightly more descriptive text than the element text content.